src/os/windows/win32.cpp

```diff
@@ -832,4 +819,6 @@ void SetWin32ThreadName(DWORD dwThreadID, const char* threadName)
 	}
 #pragma warning(pop)
 }
+#else
+void SetCurrentThreadName(const char*) {}
```

Comment:
```suggestion
void SetCurrentThreadName(const char *) {}
```
(I'm being really nitpicky now)